### DISCUSSION TITLE
🚀 Feature Request: @alova/wormhole 自定义名称生成规则能力

### DISCUSSION THREAD

**Discussion by Lnncoco** [alovajs/alova#606] [Feature Request]
### What problem does the new feature need to solve?

此前使用 swagger 生成项目 API 包，体验了下 @alova/wormhole 使用方便快捷，但引用名称上感觉有可改进的地方，希望添加一个自定义名称生成规则。
没看到有这个功能，是否考虑添加一个？
之前使用其他项目生成swagger operationId 作为名称，但麻烦问题挺多，主要还是前端无法管控，修改需要依赖后端，各种因素为了前端有一个统一美观的调用名，并不是每次都很容易。

同时还希望增加一个保存JSON文件功能，当有问题跟后端核对时需要拿JSON核对，总是使用在线的swagger存在不一致的风险。

### How should the new features you expect looks like?

如果能自定义调用名生成规则，能减少不少使用难度。
增加一个保存编译JSON的文件到本地功能。

### How important is it to you?

Can make it easier to develop

**Comment by JOU-amjs** [comment]
@Lnncoco 可以在`alova.config.[jt]s`中的`handleApi`中自定义调用名的生成，你可以自定义修改`operationId`的值。
```js
{
  // ...
  handleApi(apiDescriptor) {
    apiDescriptor.operationId = 'xxx';
    
    return apiDescriptor;
  }
}
```

另外保存编译JSON文件到本地是什么意思？出于什么情况需要这个特性呢？

**Comment by Lnncoco** [comment]
谢谢 已经用 handleApi 处理成功。

将本次编译的 swagger json 文件在当前项目里留一份。
可能使用的时候部分接口没问题，但后期发现问题，此时在线的Swagger又变更了（或者失效了七七八八原因），这时候就没地方去确认原因。查看编译后的文件无法完全确认源json就是有问题的，所以提供个保存选项在各种情况下比较方便。

**Reply by JOU-amjs** [reply]
好的，这个其实我们有计划开发一个文件变动的更新日志，用于查询每次更新的文件到底更新了哪些内容，计划在后面实现这个特性
